apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -1,11 +1,140 @@
+$small: 850px;
+
+.btn {
+  color: white;
+  text-transform: uppercase;
+  text-align: center;
+  width: 100%;
+  background-color: #D32E6A;
+  padding: 0.5em;
+  cursor: pointer;
+}
+
+.close-btn {
+  position: absolute;
+  top: 20px;
+  right: 20px;
+  z-index: 200;
+  width: 31px;
+  height: 31px;
+  cursor: pointer;
+}
+
+.reactions-container {
+  padding: 20px;
```

Comment:
Properties should be ordered position, top, left, display, width, max-width, padding, margin, transform

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)